Removing unsupported options from HIS (#975)
MicrosoftDocs/biztalk-docs: his/core/additional-help-with-events-and-errors1.md

  
4.  Double-click the event of interest.  
  

> 5.  Under **Description** see if you can investigate further.  

Reviewer: ```suggestion
5.  Under **Description**, check whether you can investigate further.  
```

> 6.  In the **Event Viewer** dialog box, click **Yes**.  

Reviewer: ```suggestion
6.  In the **Event Viewer** dialog box, select **Yes**.  
```

  
4.  Double-click an event.  
  

> 5.  Under the Description text box, see if you can investigate further.  

Reviewer: ```suggestion
5.  Under the Description text box, check whether you can investigate further.  
```

  
6.  In the **Event Viewer** dialog box, click **Yes**.  
  
    > [!NOTE]

>     >  Clicking **Yes** allows your computer to send information across the Internet.  

Reviewer: ```suggestion
    > To send information across the Internet, select **Yes** allows your computer.  
```

>     > To send information across the Internet, select **Yes** allows your computer.  

Reviewer: ```suggestion
    > To let your computer send information across the Internet, select **Yes**.  
```
